samples/enable_disable_hardware_sensors.py

```diff
@@ -0,0 +1,74 @@
+import orionsdk
+
+server = 'orion_server'
+username = 'username'
+password = 'password'
+
+# The structure of the payload required for the verbs (Enable|Disable)Sensors is not straight-forward.
+# Below is what the payload should look like:
+payload_template = [
+    {'HardwareInfoID': int, 'HardwareCategoryStatusID': int, 'UniqueName': str},
+]
+# Notice that you can supply multiple sensors in the payload.
+
+# This script shows how to create a payload to disable/enable every sensor -
+#  on a particular node by supplying the node's IP address.
+#  This could be re-written to target specific/individual sensors.
+
+
+def main():
+    ip_address = '192.168.1.1'  # Change this value to the node you want to target.
+    payload = collect_sensor_data(ip_address=ip_address)
+    change_sensor_state(payload, enabled=True)  # Change the enabled bool if you want to disable sensors.
+
+
+def sensor_payload(hardware_info_id: int, hardware_category_status_id: int, unique_name: str):
+    data = {
+        'HardwareInfoID': hardware_info_id,
+        'HardwareCategoryStatusID': hardware_category_status_id,
+        'UniqueName': unique_name
+    }
+    return data
+
+
+def change_sensor_state(payload, enabled=True):
+    # Defaults to the sensor being enabled unless enabled flag is set to False.
+    if enabled:
+        verb = 'EnableSensors'
+    else:
+        verb = 'DisableSensors'
+    client = orionsdk.SwisClient(server, username, password)
+    client.invoke('Orion.HardwareHealth.HardwareItemBase', verb, payload)
+
+
+def collect_sensor_data(ip_address):
+
+    # Hardware Sensors are tied to the node via the Node ID.
+    #  Collect the NodeID by supplying the IP address of the node.
+    client = orionsdk.SwisClient(server, username, password)
+    node_id = client.query(
+        "SELECT NodeID, IPAddress "
+        "FROM Orion.Nodes "
+        f"WHERE IPAddress = '{ip_address}' "
+    )['results'][0]['NodeID']
```

Comment:
It might be useful to add a comment to explain that this sample assumes this query will return exactly one matching record.